extension-manifest-v3/src/background/utils/map.js

```diff
@@ -69,7 +69,11 @@ export default class AutoSyncingMap {
     // and log it. A potential improvement could be to treat the
     // in-memory map as the source of truth in that scenario.)
     this._pending = new Promise((resolve, reject) => {
-      chrome.storage.local.get([this.storageKey], (result) => {
+      // Migration to session storage
+      // TODO: Remove this code after a few releases
+      chrome.storage.local.remove(this.storageKey);
+
+      chrome.storage.session.get([this.storageKey], (result) => {
```

Comment:
what's the status of chrome.storage.session on safari? do we have to bump the minimal OS requirements?

Reply:
chrome.storage.session support came with Safari 16.4 (Released 2023-03-27)

Reply:
We can make a fallback to `chrome.storage.local` if the `chrome.storage.session` is not available.